components/views/landing/Landing.styles.tsx

```diff
@@ -70,17 +67,38 @@ export const StyledLanding = styled.section`
                 h2 {
                     font-size: 3rem;
                 }
+                @media only screen and (max-width: 750px) {
```

Comment:
Same here. If multiple media queries of the same widths in one file, put them all under the same media query